ASM/c/blue_warp.c

```diff
@@ -42,6 +42,16 @@ int32_t DoorWarp1_PlayerInRange_Overwrite(z64_actor_t* actor, z64_game_t* game)
             }
             extended_savectx.collected_dungeon_rewards[boss_idx] = true;
         }
+        // set time of day each time the blue warp is taken to reduce nonrepeatable access
+        if (boss_idx == 5) { // Morpha
+            z64_file.skybox_time = z64_file.day_time = 0x4800; // CLOCK_TIME(6, 45)
```

Comment:
Why not use the CLOCK_TIME macro here instead of the straight hex values? Compiler will optimize it away anyway, but it makes the code more readable.

Reply:
The `CLOCK_TIME` macro isn't actually defined in the rando codebase. I suspect it was done this way because it didn't seem worth it to define the macro for just 8 uses (now just 2).